src/runner-command.ts

```
@@ -0,0 +1,21 @@
+  'bunx',
+  'npx',
+  'pnpm exec',
+  'yarn dlx',
+])
+
+export function formatRunnerCommand(runner: string, args: string[]): string {
+  return `${runner} ${args.join(' ')}`
+}
+
+export function splitRunnerCommand(runner: string): { command: string, args: string[] } {
+  const normalizedRunner = runner.trim().replaceAll(/\s+/g, ' ')
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)